### PR TITLE
huf-115 fix calendar clear

### PR DESCRIPTION
**Pull request by peter-subsplash**
- fix calendar clear issue (around line 3200 - changing from 2 seconds to 250ms timeout)

https://user-images.githubusercontent.com/102111077/164872759-82d51dd3-d5c4-4ce1-8cfe-570e44ec12f2.mp4

### CODE REVIEW

File: src/components/form/FluroContentFormField.vue [hunk 1]
```diff
@@ -3190,31 +3188,13 @@ export default {
...
-		//     // }
-		// },
+		}, 250),
 		toggleCollapsed(object) {
-			if (!object._collapsed) {
-				this.$set(object, '_collapsed', true);
-			} else {
-				this.$set(object, '_collapsed', false);
-			}
+			this.$set(object, '_collapsed', !object._collapsed);
```
Comment (calvinjkelly-ss) [inline src/components/form/FluroContentFormField.vue:3191]: Nice :)

File: src/components/form/FluroContentFormField.vue [hunk 2]
```diff
@@ -3168,6 +3165,7 @@ export default {
 			}
 		},
 		checkTextDateImmediate() {
+			console.log('checking datetext immediate');
```
Comment (calvinjkelly-ss) [inline src/components/form/FluroContentFormField.vue]: Curious if we need this console.log and the one on line 3191. It's fine if we do just want to make sure it wasn't left here by accident!

**Review by calvinjkelly-ss — APPROVED** [review]

File: src/components/form/FluroContentFormField.vue (hunk 2 above)
Comment (peter-subsplash) [reply src/components/form/FluroContentFormField.vue]: If I added it, it's a mistake. Will fix, thanks!

Comment (peter-subsplash) [reply src/components/form/FluroContentFormField.vue]: Ha, I had a commit hanging out on my machine clearing those.

**Pull request merged**